Orange/widgets/regression/owsgdregression.py

```diff
@@ -180,6 +162,23 @@ def _on_func_changed(self):
         for spin, enabled in zip(self._func_params, mask):
             spin.setEnabled(enabled)
 
+    def send_report(self):
+        items = OrderedDict()
+        items['Loss function'] = self.LOSS_FUNCTIONS[self.loss_function]
+        if self.loss_function != self.SqLoss:
+            items['Loss function'] +=  ", ε={}".format(self.epsilon)
```

Comment:
There is a double space after `+=`, but I'm merging anyway. Enough is enough. :)

Hint: use editors that check for PEP 8 compliance.